From 729e2b5e26309497d81accb4fcdc1ade74d10eca Mon Sep 17 00:00:00 2001
From: Robin Gong <yibin.gong@nxp.com>
Date: Mon, 27 Jun 2016 15:14:09 +0800
Subject: [PATCH 1190/5242] MLK-12928-6 mfd: pf1550: correct num_regs of
 regmap_irq_chip

commit  3821d7fdc7d76008bb84d4de991d6e69ecd62669 from
https://source.codeaurora.org/external/imx/linux-imx.git

Correct num_regs of regmap_irq_chip structure, otherwise request irq
failed.

Signed-off-by: Robin Gong <yibin.gong@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/mfd/pf1550.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mfd/pf1550.c b/drivers/mfd/pf1550.c
index a4d7f08..b60e822 100644
--- a/drivers/mfd/pf1550.c
+++ b/drivers/mfd/pf1550.c
@@ -69,7 +69,7 @@
 	.status_base		= PF1550_PMIC_REG_SW_INT_STAT0,
 	.mask_base		= PF1550_PMIC_REG_SW_INT_MASK0,
 	.mask_invert		= false,
-	.num_regs		= 4,
+	.num_regs		= 23,
 	.irqs			= pf1550_regulator_irqs,
 	.num_irqs		= ARRAY_SIZE(pf1550_regulator_irqs),
 };
-- 
1.7.9.5

